superset/assets/javascripts/explore/components/AggregateOption.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
I think it's ok to declare more than one component in a module if the component is only used once in that one module. Point being to avoid juggling with lots of small files when possible, the components can be refactored out to its own module when it grows to be used in more than one component.

I've been guilty of the other extreme  (very long modules) so I'm ok with this if you prefer.

Reply:
I do prefer more files in general.. I think in this instance it makes sense so all the `*Option` components are declared in the same way.

Reply:
++ small files. it simplifies everything vastly if everything has it's own file.